src/main/java/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -248,7 +249,14 @@ public final void createDatabase(DatabaseType dbType, Connection conn) throws SQ
             }
     }
 
-
+    public final void runScript(String scriptPath) throws SQLException, IOException {
```

Comment:
As mentioned above - separate PR with individual tests for this I think.

Reply:
For Oracle DB container spin up from Docker, some benchmarks requires gathering optimizer statistics in order to improve performance (especially tpch/chbenchmark with multiple 6+ way joins), otherwise time executed for each query would be too long. The optimizer query was invoked from this function.

Should I add test for this feature here, a separate PR, or just remove tpch + chbenchmark from GitHub Actions for Oracle?

Reply:
#397